src/components/TrendChart.js

```diff
@@ -166,6 +191,27 @@ class TrendChart extends React.Component {
                 onMouseMove={this.rememberValue}
                 onMouseOut={this.forgetValue}
               />
+              {(timeTo >= 2013 && crime === 'rape') && (
+                <g>
```

Comment:
i think it might be easier to transform this `g` element to the proper spot, and then simply place the rect and text relative to that (so that we only need to do that `x(new Date(2013-01-01)` in one spot)